examples/mlflow_artifacts/docker-compose.yml

```diff
@@ -82,6 +84,7 @@ services:
       --backend-store-uri postgresql://user:password@postgres:5432/db
       --default-artifact-root http://artifacts-server:5500/api/2.0/mlflow-artifacts/artifacts/experiments
       --gunicorn-opts "--log-level debug"
+      --serve-artifacts
```

Comment:
```suggestion
```

I think this can be removed because the client doesn't send requests for artifact operations to `tracking-server`.

Reply:
I locally tested and it worked without `--serve-artifacts`.

Reply:
removed